readme.md

```diff
@@ -30,6 +30,15 @@ getPort(3000).then(port => {
 });
 ```
 
+Additionally, you can check if a preferred port is available for a given host
+
+```js
+getPort(3000, '127.0.0.1').then(port => {
+	console.log(port);
+	// Will use 3000 if available, otherwise fall back to a random port
+});
+```
+
```

Comment:
Can you add some API docs for the API. See https://github.com/sindresorhus/boxen#api for the style to use.